src/components/CarouselItem/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+
+const CarouselItem = ({ currentIndex, length }) => {
+  return (
+    <div className="index-slide">
+      <div className="index-slide__wrapper">
+        <div className="index-slide__wrapper__current-slide">0{currentIndex}/</div>
```

Comment:
In BEM is not allowed two '__' signs in 1 style. It's a plain model, so it should be something like "index-slide__current-slide"

Reply:
ok